### PR TITLE
feat: Modal Added

### PR DESCRIPTION
**Pull request by prakash855**
Modal Added,
A modal that can be used to pop the data up or open dialogue box or open mini form!

### CODE REVIEW

**Review by Husain01 — APPROVED** [review]

File: components/Modal/Modal.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  position: fixed;
+  z-index: 1; /* Sit on top */
+  padding-top: 100px; /* Location of the box */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
```
Comment (shaik-saad) [inline components/Modal/Modal.css]: The second background line is applied here, you can remove the first one.

File: components/Modal/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+span.onclick = function() {
+  modal.style.display = "none";
+}
+
+// When the user clicks anywhere outside of the modal, close it
+window.onclick = function(event) {
+  if (event.target == modal) {
+    modal.style.display = "none";
+  }
+}
```
Comment (shaik-saad) [inline components/Modal/index.js]: you can use addEventListener here, it's much better with that and also a better practice. Also try to es6+ syntax

File: components/Modal/Modal.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+/* Modal Content */
+.modal-content {
+  background-color: #fefefe;
```
Comment (NamrataRaikwar2002) [inline components/Modal/Modal.css]: You should take colors in variables.

File: index.js [hunk 4]
```diff
@@ -27,4 +27,4 @@ inputText.addEventListener("input", (e) => {
 // Case TransformStream in the Text Utilities
 
 document.getElementById('lowerCased').innerText.toUpperCase()
-document.getElementById('upperCased').innerText.toLowerCase()
\ No newline at end of file
+document.getElementById('upperCased').innerText.toLowerCase()
```
Comment (shaik-saad) [inline index.js:30]: you can use css text-transform property instead for this js functionality

**Review by shaik-saad — APPROVED** [review]
Added few suggestion, do check them out

**Review by NamrataRaikwar2002 — APPROVED** [review]
Looks good, gave some suggestions.

File: components/Modal/Modal.css (hunk 3 above)
Comment (prakash855) [reply components/Modal/Modal.css]: Yeah sure, thanks!

File: components/Modal/index.js (hunk 2 above)
Comment (prakash855) [reply components/Modal/index.js]: sure, thanks

File: index.js (hunk 4 above)
Comment (prakash855) [reply index.js:30]: Sure, will do that

**Pull request merged**